client/a8c-for-agencies/data/purchases/lib/format-licenses.ts

```diff
@@ -38,5 +46,20 @@ export default function formatLicenses( items: APILicense[] ): License[] {
 		quantity: item.quantity,
 		parentLicenseId: item.parent_license_id,
 		referral: item.referral,
+		meta: formatLicenseMeta( item.meta ),
 	} ) );
 }
+
+export function formatLicenseMeta( meta: APILicenseMeta = {} ): LicenseMeta {
+	const isDevSite = meta?.a4a_is_dev_site === '1' ? true : false;
```

Comment:
I believe there is no need for the ternary operator here. 
`meta?.a4a_is_dev_site === '1'` will always return a boolean. Same goes for the line below.